### PR TITLE
some more tiny fixes

### PR DESCRIPTION
**Pull request by tshepang**

### CODE REVIEW

File: _posts/2015-05-08-rustful-0-1-0.md [hunk 1]
```diff
@@ -323,7 +323,7 @@ are demonstrated.
 
 ## So, What Happens Next?
 
-This is only the beginning, and there are still a things to do (especially
+This is only the beginning, and there are still a lot of things to do (especially
```
Comment (Ogeon) [inline _posts/2015-05-08-rustful-0-1-0.md:326]: I wanted to change it to "and there are still things to do", but it looks like I left a rouge "a" in there. I'm actually not sure what sounds best...

Comment (tshepang) [inline _posts/2015-05-08-rustful-0-1-0.md:326]: the things to do are "a lot" anyways, right :)

**Comment by Ogeon** [comment]
Thank you so much for this! Proof reading your own text is hard and it becomes even harder if English isn't your primary language... Have you read the whole post or can I expect more? :)

File: _posts/2015-05-08-rustful-0-1-0.md (hunk 1 above)
Comment (Ogeon) [inline _posts/2015-05-08-rustful-0-1-0.md:326]: Haha, I suppose that's true :)

**Comment by tshepang** [comment]
I've read the whole post (sort of), so don't expect more.

**Comment by tshepang** [comment]
And I think your writing is good, but I agree that it's hard to write something that lacks defects.

**Comment by Ogeon** [comment]
Alright, thanks again. I'm usually not that worried when writing comments like this one, but writing such a thing as this post made me a bit nervous and I don't know how many times I rewrote some parts. Anyway, I hope you found it interesting.

**Pull request merged**